path_planning/src/path_planning/state_tree.py

```diff
@@ -1,42 +1,51 @@
 # tree data structure
 from graphviz import Digraph, Source
+import rospkg
 
 
 class Tree:
     """
-        A class to represent the structure of a node
-
-        ...
-
-        Attributes
-        ----------
-        name : str
-            Name of the state.
-        nodeType : str
-            Type of state machine - state, parallel state machine, sequential state machine.
-        children : list
-            A list of the children, or substates, of the state.
-        daemon : list
-            A list of the UNIMPORTANT children, or substates, of the state.
-        depth : int
-            Represents the depth of the node of the tree within the state machine.
-        number : int
-            A unique integer value assigned to each state to easily differentiate between states.
-
-        Methods
-        -------
-        renderTree(path : str):
-            Renders the structure of the state machine tree to the specified path in the form of a .pdf file.
-
-        renderGraph(path : str):
-            Renders the flowchart of the state machine to the specified path in the form of a .pdf file.
-
-        size() -> int:
-            returns the number of nodes within the state machine
-        """
+    A class to represent the structure of a node
```

Comment:
Didn't change any of the documentation, just replaced tabs with spaces to get rid of linting errors